metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Manages client.rb configuration and chef-client service'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.0.0'
+version           '4.0.1'
```

Comment:
Please don't bump the version in pull requests. The maintainers will handle versioning.